features/step_definitions/cluster_configuration/localstorage.rb

```diff
@@ -1,245 +1,95 @@
+    })
+    raise "Create subscription #{name} failied" unless @result[:success]
   end
-  File.write(filepath, cfm.to_yaml)
-  res = admin.cli_exec(:create, n: namespace, f: filepath)
-  raise "error creating configmap for local storage provisioner" unless res[:success]
-
-  step %Q/I switch to cluster admin pseudo user/
-  sa = service_account(serviceaccount)
-  @result =  sa.create(by: admin)
-  step %Q/the step should succeed/
-
-  _opts = {scc: "privileged", user_name: "system:serviceaccount:#{namespace}:#{serviceaccount}"}
-  add_command = :oadm_policy_add_scc_to_user
-  @result = admin.cli_exec(add_command, **_opts)
-  step %Q/the step should succeed/
 
-  step %Q/I run the :create admin command with:/, table(%{
-      | f | https://raw.githubusercontent.com/openshift/origin/release-3.9/examples/storage-examples/local-examples/local-storage-provisioner-template.yaml |
-      | n | #{namespace}                                                                                                                                    |
+  # check status
+  step %Q/a pod becomes ready with labels:/, table(%{
+    | name=local-storage-operator |
   })
-  step %Q/the step should succeed/
-
-  step 'admin ensures "local-storage:provisioner-node-binding" clusterrolebinding is deleted'
-  step 'admin ensures "local-storage:provisioner-pv-binding" clusterrolebinding is deleted'
-  step %Q/I run the :new_app admin command with:/, table(%{
-      | param    | CONFIGMAP=#{configmap}            |
-      | param    | SERVICE_ACCOUNT=#{serviceaccount} |
-      | param    | NAMESPACE=#{namespace}            |
-      | param    | PROVISIONER_IMAGE=#{image}        |
-      | template | #{template}                       |
-      | n        | #{namespace}                      |
-  })
-  step %Q/the step should succeed/
-
-  nodes = env.nodes.select { |n| n.schedulable? }
-  step %/#{nodes.size} pods become ready with labels:/, table(%{
-      | app=local-volume-provisioner|
-  })
-
-  pv_count = 0
-  BushSlicer::PersistentVolume.list(user: admin).each { |pv|
-    if pv.name.start_with?("local-pv-") && (pv.local_path&.start_with?("#{path}/fast") || pv.local_path&.start_with?("#{path}/slow"))
-      pv_count += 1
-    end
-  }
-
-  raise "error creating PVs with local storage provisioner" unless (pv_count == nodes.size * 4)
-
-  lf_sc = storage_class("local-fast")
-  lf_sc.ensure_deleted(user: admin)
-  ls_sc = storage_class("local-slow")
-  ls_sc.ensure_deleted(user: admin)
-
-  step %Q{I obtain test data file "storage/misc/storageClass.yaml"}
-  sc = YAML.load(@result[:response])
-  filepath = @result[:abs_path]
-  sc["metadata"]["name"] = "local-fast"
-  sc["provisioner"] = "kubernetes.io/no-provisioner"
-  sc["volumeBindingMode"] = "Immediate"
-  File.write(filepath, sc.to_yaml)
-  @result = admin.cli_exec(:create, f: filepath)
-  step %Q/the step should succeed/
+  raise "No pod is running for local storage operator" unless @result[:success]
```

Comment:
This check is not needed.  As above check will fail itself if the criteria is not met.

Reply:
Fixed.